app/src/main/java/sdp/moneyrun/ui/authentication/SignUpActivity.java

```diff
@@ -34,7 +35,11 @@ protected void onCreate(Bundle savedInstanceState) {
             EditText passwordView = findViewById(R.id.signUpPassword);
             String email = emailView.getText().toString().trim();
             String password = passwordView.getText().toString().trim();
-
+            isClicked = !isClicked;
```

Comment:
Are you sure you can do this before initializing isClicked ? 
I just tried in a small test class and it doesnt compile.

Reply:
isClicked is initialized to false by default, but it might be cleaner to do an explicit assignation

Reply:
I know. That's why I did my little test appart, even though it gets initialized to false by default, doing it this way this seems to still prouduce a nullpointer exception bug cause it wasnt initialized. I invite you to create a small main class somewhere and test it out if you have any doubts

Reply:
Weird, in the app it seems to be working perfectly without initialization but I put it just in case thanks for pointing it out if you found an edge case where this can potentially cause a problem